src/compiler/diagnosticMessages.json

```diff
@@ -6276,6 +6276,11 @@
         "category": "Error",
         "code": 8034
     },
+    "You cannot rename elements that are defined in an external library inside `node_modules/`.": {

```

Comment:
Unless we use it elsewhere, I'm not sure the meaning of "external library" will be clear.  Personally, I'd probably split this into two messages along the lines of "you can't reach into node_modules" and "you can't reach into _other_ node_modules"